content/en/docs/tutorial/microservices-istio/single/index.md

```diff
@@ -0,0 +1,129 @@
+on a local machine. You don't need to write code though. Instead, you build,
+run, and test an existing service: `ratings`.
+
+The `ratings` service is a small web app written in
+[Node.js](https://nodejs.org/en/) that can run on its own. It performs similar
+actions to those of other web apps:
+
+- Listen to the port it receives as a parameter.
+- Expect `HTTP GET` requests on the `/ratings/{productID}` path and return the
+  ratings of the product matching the value you specify for `productID`.
+- Expect `HTTP POST` requests on the `/ratings/{productID}` path and update the
+  ratings of the product matching the value you specify for `productID`.
+
+Follow these steps to download the code of the app, install its dependencies,
+and run it locally:
+
+1. Download
+    [the service's code]({{< github_blob >}}/samples/bookinfo/src/ratings/ratings.js)
+    and
+    [the package file]({{< github_blob >}}/samples/bookinfo/src/ratings/package.json)
+    into a separate directory:
+
+    {{< text bash >}}
+    $ mkdir ratings
+    $ cd ratings
+    $ curl -s {{< github_file >}}/samples/bookinfo/src/ratings/ratings.js -o ratings.js
+    $ curl -s {{< github_file >}}/samples/bookinfo/src/ratings/package.json -o package.json
+    {{< /text >}}
+
+1. Skim the service's code and note the following elements on the code:
+    - The web server's features:
+        - listening to a port
+        - handling requests and responses
+    - The aspects related to HTTP:
+        - headers
+        - path
+        - status code
+
+    {{< tip >}}
+    In Node.js, the web server's functionality is embedded in the code of the application. A Node.js
+    web application runs as a standalone process.
+    {{< /tip >}}
+
+1. Node.js applications are written in JavaScript, which means that there is no
+    explicit compilation step. Instead, they use [just-in-time compilation](https://en.wikipedia.org/wiki/Just-in-time_compilation). To build a Node.js application, then means to install its dependencies. Install
+    the dependencies of the `ratings` service in the same folder where you
+    stored the service code and the package file:
+
+    {{< text bash >}}
+    $ npm install
+    npm notice created a lockfile as package-lock.json. You should commit this file.
+    npm WARN ratings No description
+    npm WARN ratings No repository field.
+    npm WARN ratings No license field.
+
+    added 24 packages in 2.094s
+    {{< /text >}}
+
+1. Run the service, passing `9080` as a parameter. The application then listens on port 9080.
+
+    {{< text bash >}}
+    $ npm start 9080
+    > @ start /tmp/ratings
+    > node ratings.js "9080"
+    Server listening on: http://0.0.0.0:9080
+    {{< /text >}}
+
+{{< tip >}}
+The `ratings` service is a web app and you can communicate with it as you would
+with any other web app. You can use a browser or a command line web client like
+[cURL](https://curl.haxx.se) or [Wget](https://www.gnu.org/software/wget/).
```

Comment:
Shouldn't this be `curl` and `wget`? Prevents the ned for the dictionary add for the second.

Reply:
Fixed curl, but it is actually Wget with capital W so I fixed that one differently than spelling file  by using the backtick syntax we provide for this kind of thing.